examples/react-native-example/ios/MeshConnectRNSDK/Info.plist

```diff
@@ -26,22 +26,19 @@
 	<true/>
 	<key>NSAppTransportSecurity</key>
 	<dict>
-		<key>NSExceptionDomains</key>
-		<dict>
-			<key>localhost</key>
-			<dict>
-				<key>NSExceptionAllowsInsecureHTTPLoads</key>
-				<true/>
-			</dict>
-		</dict>
+	  <!-- Do not change NSAllowsArbitraryLoads to true, or you will risk app rejection! -->
+		<key>NSAllowsArbitraryLoads</key>
+		<false/>
+		<key>NSAllowsLocalNetworking</key>
+		<true/>
```

Comment:
doesn't setting `NSAllowsArbitraryLoads` to false and not having `NSExceptionDomains` mean the SDK will not be able to load content ?

Reply:
This is in the example folder which is not SDK code. That's why I set them like this.